README.md

```diff
@@ -59,6 +61,8 @@ assets:
     automatic_img_alt : true | false | default: true
 ```
 
+Read more about [Jekyll configuration](https://jekyllrb.com/docs/configuration/).
```

Comment:
Jekyll needs to handle it's own documentation, we don't need to link to their documentation, this is about Jekyll Assets not Jekyll.  I understand it's a fair point to link to Jekyll, but that's about it, not to Jekyll's configuration, that's a mixed concern.